packages/form-footer/src/FormFooter.tsx

```diff
@@ -141,23 +192,41 @@ export default function FormFooter({
   ...rest
 }: FormFooterProps) {
   const { theme, darkMode } = useDarkMode(darkModeProp);
+
+  const backButton = {
```

Comment:
nit: backButtonProps (or similar. It's not the button itself)

Reply:
ahh yeah did this to avoid naming collision but I can rename to something else